app/components/createContainer/createcontainer.html

```diff
@@ -295,7 +295,7 @@
                 <div class="col-sm-9">
                   <select class="form-control" ng-model="config.HostConfig.NetworkMode" id="container_network">
                     <option selected disabled hidden value="">Select a network</option>
-                    <option ng-repeat="net in availableNetworks" ng-value="net.Name">{{ net.Name }}</option>
+                    <option ng-repeat="net in availableNetworks" ng-value="net.Id">{{ net.Name }}</option>
```

Comment:
Not sure about this as some networks are hard-coded using only names at the following locations:

* https://github.com/portainer/portainer/blob/develop/app/components/createContainer/createContainerController.js#L32
* https://github.com/portainer/portainer/blob/develop/app/components/createContainer/createContainerController.js#L98-L105

This might introduce some bugs as containers would sometimes be created using network by name and sometimes network by ID. 

Another minor thing is that the network dropdown now defaults to an empty value as it's not capable to map the default value of `bridge` defined here https://github.com/portainer/portainer/blob/develop/app/components/createContainer/createContainerController.js#L32 to an ID.